generator-java/lib/assert/internal/assert.liberty.js

```diff
@@ -78,15 +77,14 @@ test_liberty.prototype.assertFeatures = function() {
   if(arguments.length < 1) {
     throw "assertFeatures error : requires at least 1 argument, a feature to check";
   }
-  var i;
-  var check;
-  var exists = true;
+  let i;
+  let exists = true;
   if(typeof(arguments[0]) === "boolean") {
     exists = arguments[0];
-    check = arguments[0] ? assert.fileContent : assert.noFileContent;
+    if (arguments[0])
+      assert.fileContent
     i = 1;
```

Comment:
I think we can remove the assert.fileContent completely, because that line doesn't actually call the function